### ISSUE TITLE
django.core.exceptions.ImproperlyConfigured: allauth.socialaccount not installed, yet its models are imported.

### ISSUE THREAD

**Issue by half-of-a-glazier** [pennersr/django-allauth#3824]
I upgraded django-allauth from 0.61.1 to 0.63.0 and this error popped up when I tried logging in. Mfa is required but not activated yet. Socialaccount isn't installed because I'm not using it in my app.
Update: When I downgraded back to 0.61.0 the error disappeared.

```Traceback (most recent call last):
  File "c:\Users\user\Documents\Integralytic\Capstone Care\CapstoneBackend\.venv\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "c:\Users\user\Documents\Integralytic\Capstone Care\CapstoneBackend\.venv\lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "c:\Users\user\Documents\Integralytic\Capstone Care\CapstoneBackend\.venv\lib\site-packages\django\views\generic\base.py", line 104, in view
    return self.dispatch(request, *args, **kwargs)
  File "c:\Users\user\Documents\Integralytic\Capstone Care\CapstoneBackend\.venv\lib\site-packages\django\utils\decorators.py", line 48, in _wrapper
    return bound_method(*args, **kwargs)
  File "c:\Users\user\Documents\Integralytic\Capstone Care\CapstoneBackend\.venv\lib\site-packages\allauth\decorators.py", line 12, in wrap
    resp = function(request, *args, **kwargs)
  File "c:\Users\user\Documents\Integralytic\Capstone Care\CapstoneBackend\.venv\lib\site-packages\django\utils\decorators.py", line 48, in _wrapper
    return bound_method(*args, **kwargs)
  File "c:\Users\user\Documents\Integralytic\Capstone Care\CapstoneBackend\.venv\lib\site-packages\django\views\decorators\debug.py", line 141, in sensitive_post_parameters_wrapper
    return view(request, *args, **kwargs)
  File "c:\Users\user\Documents\Integralytic\Capstone Care\CapstoneBackend\.venv\lib\site-packages\django\utils\decorators.py", line 48, in _wrapper
    return bound_method(*args, **kwargs)
  File "c:\Users\user\Documents\Integralytic\Capstone Care\CapstoneBackend\.venv\lib\site-packages\django\views\decorators\cache.py", line 80, in _view_wrapper
    response = view_func(request, *args, **kwargs)
  File "c:\Users\user\Documents\Integralytic\Capstone Care\CapstoneBackend\.venv\lib\site-packages\allauth\account\views.py", line 85, in dispatch
    return super().dispatch(request, *args, **kwargs)
  File "c:\Users\user\Documents\Integralytic\Capstone Care\CapstoneBackend\.venv\lib\site-packages\allauth\account\mixins.py", line 40, in dispatch
    response = super().dispatch(request, *args, **kwargs)
  File "c:\Users\user\Documents\Integralytic\Capstone Care\CapstoneBackend\.venv\lib\site-packages\django\views\generic\base.py", line 143, in dispatch
    return handler(request, *args, **kwargs)
  File "C:\Users\user\Documents\Integralytic\Capstone Care\CapstoneBackend\capstoneApp\views\allauth_views.py", line 47, in post
    response = self.form_valid(form)
  File "c:\Users\user\Documents\Integralytic\Capstone Care\CapstoneBackend\.venv\lib\site-packages\allauth\account\views.py", line 98, in form_valid
    return form.login(self.request, redirect_url=redirect_url)
  File "c:\Users\user\Documents\Integralytic\Capstone Care\CapstoneBackend\.venv\lib\site-packages\allauth\account\forms.py", line 198, in login
    ret = flows.login.perform_password_login(request, credentials, login)
  File "c:\Users\user\Documents\Integralytic\Capstone Care\CapstoneBackend\.venv\lib\site-packages\allauth\account\internal\flows\login.py", line 30, in perform_password_login
    return perform_login(request, login)
  File "c:\Users\user\Documents\Integralytic\Capstone Care\CapstoneBackend\.venv\lib\site-packages\allauth\account\internal\flows\login.py", line 39, in perform_login
    return resume_login(request, login)
  File "c:\Users\user\Documents\Integralytic\Capstone Care\CapstoneBackend\.venv\lib\site-packages\allauth\account\internal\flows\login.py", line 48, in resume_login
    response = ctrl.handle()
  File "c:\Users\user\Documents\Integralytic\Capstone Care\CapstoneBackend\.venv\lib\site-packages\allauth\account\stages.py", line 96, in handle
    unstash_login(self.request)
  File "c:\Users\user\Documents\Integralytic\Capstone Care\CapstoneBackend\.venv\lib\site-packages\allauth\account\utils.py", line 174, in unstash_login
    login = Login.deserialize(data)
  File "c:\Users\user\Documents\Integralytic\Capstone Care\CapstoneBackend\.venv\lib\site-packages\allauth\account\models.py", line 278, in deserialize
    from allauth.socialaccount.models import SocialLogin
  File "c:\Users\user\Documents\Integralytic\Capstone Care\CapstoneBackend\.venv\lib\site-packages\allauth\socialaccount\models.py", line 26, in <module>
    raise ImproperlyConfigured(
django.core.exceptions.ImproperlyConfigured: allauth.socialaccount not installed, yet its models are imported.
```

**Comment by pennersr** [comment]
Fixed via 6da7730a

**Comment by pennersr** [comment]
https://codeberg.org/allauth/django-allauth/issues/3824
